applications/HDF5Application/python_scripts/point_set_output_process.py

```diff
@@ -0,0 +1,144 @@
+import KratosMultiphysics
+import KratosMultiphysics.HDF5Application as HDF5Application
+from KratosMultiphysics.HDF5Application.core.operations.model_part import Prefix
+
+
+def Factory(parameters: KratosMultiphysics.Parameters,
+            model: KratosMultiphysics.Model):
+    if not isinstance(parameters, KratosMultiphysics.Parameters):
+        raise Exception("expecting input parameters of type KratosMultiphysics.Parameters, but got {}".format(type(parameters)))
+    return PointSetOutputProcess(model, parameters)
+
+
+
+class PointSetOutputProcess(KratosMultiphysics.OutputProcess):
+
+    def __init__(self, model: KratosMultiphysics.Model, parameters: KratosMultiphysics.Parameters):
+        KratosMultiphysics.OutputProcess.__init__(self)
+        parameters.ValidateAndAssignDefaults(self.GetDefaultParameters())
+        parameters["file_parameters"].ValidateAndAssignDefaults(self.GetDefaultParameters()["file_parameters"])
+
+        self.model_part = model.GetModelPart(parameters["model_part_name"].GetString())
+        self.variables = parameters["output_variables"].Clone()
+
+        configuration_name = parameters["search_configuration"].GetString()
+        if configuration_name == "initial":
+            search_configuration = KratosMultiphysics.Configuration.Initial
+        elif configuration_name == "current":
+            search_configuration = KratosMultiphysics.Configuration.Current
+        else:
+            raise RuntimeError("Invalid search configuration: '{}'".format(configuration_name))
+
+        search_tolerance = parameters["search_tolerance"].GetDouble()
+        locator = HDF5Application.BruteForcePointLocatorAdaptor(self.model_part,
+                                                                search_configuration,
+                                                                search_tolerance)
+
+        self.interval_utility = KratosMultiphysics.IntervalUtility(parameters)
+        self.output_frequency = parameters["output_frequency"].GetInt()
+        self.output_counter = 0
+
+        self.file_parameters = parameters["file_parameters"].Clone()
+
+        self.isMPIRun = self.model_part.IsDistributed()
```

Comment:
```suggestion
        self.is_distributed = self.model_part.IsDistributed()
```

it might be that Kratos runs in MPI but a solver runs in serial